.github/workflows/build-test.yaml

```diff
@@ -46,7 +54,67 @@ env:
   TEST_LEVEL: 1            # for stack tests
 
 jobs:
+
+  fourmolu:
```

Comment:
In the README we have
```
  see the file [.github/workflows/rustfmt.yaml](.github/workflows/rustfmt.yaml),
  look for `nightly-...`).
```

please fix that now that the file is gone and the job is moved.